osgp/protocol-adapter-dlms/osgp-protocol-simulator-dlms/simulator/dlms-device-simulator/src/main/resources/application-e650.properties

```diff
@@ -1,6 +1,6 @@
 default.clock.year=2015
 default.clock.month=8
-default.clock.dayOfMonth=2
+default.clock.dayOfMonth=1
```

Comment:
Moet in deze properties ook de default.clock.timezone=UTC toegevoegd worden?

Reply:
Inderdaad zou hier ook de timezone gezet kunnen worden, maar dan op Amsterdam

Reply:
Aangepast en ingecheckt